src/SqlSquared/Signature.purs

```diff
@@ -528,11 +528,11 @@ genFunctionDecl n = do
 genImport ∷ ∀ m a. Gen.MonadGen m ⇒ MonadRec m ⇒ m (SqlDeclF a)
 genImport = map Import Pt.genAnyDirPath
 
-genIdent ∷ ∀ m. Gen.MonadGen m ⇒ m String
+genIdent ∷ ∀ m. Gen.MonadGen m ⇒ m ID.Ident
```

Comment:
This should live in `SqlSquared.Signature.Ident` I think.

Reply:
Makes sense 👍